matRadGUI.m

```diff
@@ -26,13 +26,13 @@
 
```

Comment:
Why the hack do we have all these changes with regard to white spaces? This is very annoying. Where is this coming from? Can you change this?